.ci/load/Jenkinsfile

```diff
@@ -22,6 +22,7 @@ pipeline {
         timestamps()
         ansiColor('xterm')
         durabilityHint('PERFORMANCE_OPTIMIZED')
+        skipDefaultCheckout() // don't re-checkout at avery stage
```

Comment:
👍 This is a really good idea!

Reply:
I still need to test with production setup, but I think that not doing this makes the "clean and wipe workspace" also applied on every single stage and only once at the beginning of the pipeline execution.